src/ripple/nodestore/impl/Shard.h

```diff
@@ -296,7 +296,7 @@ class Shard final
     // NuDB key/value store for node objects
     std::unique_ptr<Backend> backend_;
 
-    std::atomic<std::uint32_t> backendCount_{0};
+    std::uint32_t backendCount_{0};
```

Comment:
Doesn't `backendCount_` still need to be `atomic` because the `Shard::Count` destructor can decrement `backendCount_` outside the lock?